spec/support/capybara.rb

```diff
@@ -1,8 +1,12 @@
-require "capybara/poltergeist"
+require "selenium/webdriver"
```

Comment:
I think this is technically not needed, since it is auto-required via the Gemfile.

Reply:
Fails without the require.